CFEScraping/index.js

```diff
@@ -0,0 +1,30 @@
+try {
```

Comment:
All the file is asynchronous? No , so we don't need cover all the file with the try-catch , only used on request or asycronus flows